Create garden (#3)
SkyKings-Guild/Guides: guides/garden.md

```yaml {metadata}
title: The Ultimate Garden Guide
description: |
  Learn how to play the garden update.
category: Other
author: djjxx
tags:
  - Farming
  - Garden
  - Money Making Method
```

Garden is a great money making method and also a great way to earn skyblock xp. This guide talks about how to level up 
your garden level, what to do with visitors and good money making methods

> Part 1: Leveling up your Garden

Reviewer: ```suggestion
## Part 1: Leveling up your Garden
```

> To level up your garden you need to one, unlock crop milestones, or two, complete visitor offers. At the start, you should

Reviewer: ```suggestion
To level up your garden you need to either unlock crop milestones or complete visitor offers. At the start, you should
```

get your crop milestones to at least 16 in all of them. I suggest you build a mega farm so you can farm as many crops at a
time. 

> Part 1.5: Visitors

Reviewer: ```suggestion
### Visitors
```

> With visitors, you should accept all of the unique visitors, thevisitors with cheep offers, legendarys (they are 

Reviewer: ```suggestion
With visitors, you should accept all of the unique visitors, visitors with cheap offers, legendaries (they are 
```

> profit almost all of the time) and visitors that have crops you can farm.Getting to garden 10 is pretty easy but every 

Reviewer: ```suggestion
profit almost all of the time) and visitors that have crops you can farm. Getting to garden 10 is pretty easy but every 
```

level after 10 needs 10k garden xp to level up.

> Part 2: Money Making Methods

Reviewer: ```suggestion
## Part 2: Money Making Methods
```

There are many good money making methods and some of them do require luck. The first money making method is farming in
general. The best crop to farm is nether wart with a mooshroom cow as you get a lot from the nether wart and it has a 

> decent price. Other crops still make a lot of money but nether wart is the best The second money making method is

Reviewer: ```suggestion
decent price. Other crops still make a lot of money but nether wart is the best.

The second money making method is
```

getting lucky with visitors. Visitors have a very small chance to drop an overgrown grass and a green bandana, each

> going for decently high prices. The third and final money making method of this guide is farming mushrooms for

Reviewer: ```suggestion
going for decently high prices. 

The third and final money making method of this guide is farming mushrooms for
```

burrowing spores. Burrowing spores are slowly going down in price, but with the mushrooms you farm and a little
bit of luck you can get lucky with it. Farming mushrooms is also a great way to gain farming xp.

> Part 3: The Composter

Reviewer: ```suggestion
## The Composter
```
